### ISSUE TITLE
Where is the VLM-tuning model (VideoChat finetuned using human annotation)?

### ISSUE THREAD

**Issue by linzhiqiu** [Vchitect/VBench#43]

**Comment by yinanhe** [comment]
Hello, thanks for your interest in our work.

Since `VideoChat` has been further updated for several versions now, we recommend you to use the newer version VideoChat2.

You can get the latest VideoChat2 model through [ask-anything github](https://github.com/OpenGVLab/Ask-Anything/tree/main/video_chat2), download our [human-preference data](https://drive.google.com/drive/folders/1on66fnZ8atRoLDimcAXMxSwRxqN8_0yS), and finetune on the pretrained [chat model](https://huggingface.co/OpenGVLab/VideoChat2_stage3_Mistral_7B) with a small learning rate using the [training script](https://github.com/OpenGVLab/Ask-Anything/blob/main/video_chat2/scripts/videochat_mistral/run_7b_stage3.sh).

**Comment by ziqihuangg** [comment]
Hi, we're closing the issue since there is no follow-up question, but feel free to re-open it if you have any further questions!
